conversion/data_from_database.go

```diff
@@ -221,4 +270,20 @@ func (dd *DataFromDatabaseImpl) dataFromDatabaseForBulkMigration(sourceProfile p
 	}
 
 	return bw, nil
-}
\ No newline at end of file
+}
+
+func GetBucket(project, location, profileName string) (string, string, error) {
+	ctx := context.Background()
+	dsClient, err := datastreamclient.NewDatastreamClientImpl(ctx)
+	if err != nil {
+		return "", "", fmt.Errorf("datastream client can not be created: %v", err)
+	}
+	// Fetch the GCS path from the destination connection profile.
+	dstProf := fmt.Sprintf("projects/%s/locations/%s/connectionProfiles/%s", project, location, profileName)
+	res, err := dsClient.GetConnectionProfile(ctx, dstProf)
+	if err != nil {
+		return "", "", fmt.Errorf("could not get connection profile: %v", err)
+	}
+	gcsProfile := res.Profile.(*datastreampb.ConnectionProfile_GcsProfile).GcsProfile
+	return gcsProfile.GetBucket(), gcsProfile.GetRootPath(), nil
+}
```

Comment:
Is this refactored or newly added code? 
Looks like something should be in `datastream_accessor`?

Reply:
This is existing code refactored to use the accessor. Didn't make any other changes except use of accessor in it as it is used in a few functions in UI as well